pkg/services/object/put/distributed.go

```diff
@@ -128,6 +132,10 @@ loop:
 	}
 
 	if !traverser.Success() {
+		if err, ok := resErr.Load().(error); ok {
+			return nil, err
```

Comment:
Maybe wrap it around `errIncompletePut`? This is useful error, node tried all placements and haven't succeed, but requires a bit more details from `resErr` if they are available.